docs/management/advanced-options.asciidoc

```diff
@@ -454,6 +454,7 @@ of buckets to try to represent.
 
 [horizontal]
 [[visualization-colormapping]]`visualization:colorMapping`::
+**Deprecated**
```

Comment:
Please add more informative message like e.g. 

**This setting is deprecated and will no longer be supported as of 8.0**

Reply:
@flash1293 do you think this is enough or should we add more details?

Reply:
Hey, I would like to add a note here about this setting only applying to Visualize charts and TSVB (not Lens). @gchaps do you have an idea how to word this?

Reply:
I updated text for deprecation.